doc/Language/objects.pod6

```diff
@@ -175,9 +172,9 @@ Now, after a C<Journey> object is created, its C<.origin>, C<.destination>,
 and C<.notes> will all be accessible from outside the class, but only
 C<.notes> can be modified.
 
-The above allows creation of C<Journey> objects without an origin or
-destination. To change this you can provide a default value or make
-sure that it is provided by marking it with L<is required>.
+The above allows creation of C<Journey> objects without origin or
```

Comment:
How about:
`If an object is instantiated without certain attributes, such as origin or destination, above, we may not get the desired result. To prevent this, provide default values or make sure that an attribute is set on object creation by marking an attribute with an L<is required> trait.`